client/me/notification-settings/settings-form/actions.jsx

```diff
@@ -35,11 +35,11 @@ class NotificationSettingsFormActions extends React.PureComponent {
 						onClick={ this.props.onSaveToAll }
 						isPrimary={ false }
 					>
-						{ this.props.translate( 'Save to All Sites' ) }
+						{ this.props.translate( 'Save to all sites' ) }
 					</FormButton>
 				) }
 				<FormButton disabled={ this.props.disabled } onClick={ this.props.onSave }>
-					{ this.props.translate( 'Save Settings' ) }
+					{ this.props.translate( 'Save settings' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **95** times:
`translate( 'Save Settings' )` *ES Score: 8*